images/dcos/root/etc/nginx/conf.d/default.conf.template

```diff
@@ -0,0 +1,23 @@
+server {
+    listen <PORT>;
+    server_name "";
+
+    root /usr/html;
+    index index.html;
+
+    location = / {
+        return 301 $scheme://$http_host/api/v1/proxy/namespaces/kube-system/services/kube-ui/;
+    }
+
```

Comment:
don't we need a separate rule for `/api`? I don't have nginx experience and I'm confused as to how this config actually works. Does this say that: (a) when the full URL path is `/` then send a redirect; (b) proxy everything else under `/` to APISERVER, other than for the rules that follow?

If so, some comments would be nice